vtex-apps/address-locator
# feat(autocomplete): change from search box to autocomplete component

#### What is the purpose of this pull request?

We were using the Google's SearchBox, when we should be using Autocomplete. We can now restrict our search for a specific country and for only full addresses.

#### What problem is this solving?

Now when typing "botafogo" it will appear only full addresses that contain "Botafogo"

#### How should this be manually tested?

Type "Botafogo" on the first box (or a city name) and see only "Praia de botafogo" appear on the first option

#### Screenshots or example usage

<img width="871" alt="screen shot 2018-11-26 at 15 51 52" src="https://user-images.githubusercontent.com/4925068/49032254-45e62f80-f193-11e8-8f30-d4b5b606f8a3.png">
<img width="846" alt="screen shot 2018-11-26 at 15 52 03" src="https://user-images.githubusercontent.com/4925068/49032257-45e62f80-f193-11e8-9625-3c3681083015.png">


#### Types of changes


EDIT: still need to change the suffix button, still using the location icon for now, need a designer opinion
